apps/frontend/src/assets/css/scaffolding.css

```diff
@@ -31,3 +34,38 @@ body {
 	white-space: nowrap;
 	border: 0;
 }
+
+/* circles */
+.circle {
+	width: 30px;
+	height: 30px;
+	border-radius: 100%;
+}
+
+.circle-blue {
+	background: var(--primary-gradient);
+}
+
+.circle-pink {
+	background: var(--pink-gradient);
+}
+
+/* buttons */
+.btn {
+	width: 100%;
+	padding: 10px 0;
+	font-family: var(--nunito-font);
+	font-size: 16px;
+	font-weight: 700;
+	color: white;
+	border: none;
+	border-radius: 32px;
+}
+
+.btn-dark {
+	background-color: var(--dark);
+}
+
+.btn:disabled {
+	background-color: #e0e0e0;
```

Comment:
Maybe create a css variable for #e0e0e0?

Reply:
oke, i add variabel for this color ang given name "disable-color"